README.md

```diff
@@ -1,27 +1,36 @@
-# Project Name
+# FCC FERRIS WHEEL
 
-> One paragraph statement about the project.
+> A ferris wheel webpage using animate elements, transform and speed.
 
-![screenshot](./app_screenshot.png)
-
-Additional description about the project and its features.
+![screenshot](./_C__projects_fcc_fcc-ferris-wheel_index.html.png)
 
 ## Built With
 
-- Major languages,
-- frameworks,
-- technologies used
+- HTML5 elements
+  - Block-level elements such as `div`.
+- CSS properties
+  - CSS Positioning `absolute`,
+  - CSS transform such as `rotate`,
+  - CSS Variables for reusable CSS values,
+  - CSS speed adjust such as `infinite`, `10s`, and so on,
+  - CSS animate such as `cabins`, `ease-in-out`, and so on,
```

Comment:
I suggest you make use of the CSS properties for animations here rather than the values

Reply:
Done with that now.